src/assets/_responsive.scss

```diff
@@ -16,3 +16,8 @@
   }
 }
 
+@mixin lg-down {
```

Comment:
Consider changing all the breakpoints to specify either only `max-width` and write default styles with big screens in mind using breakpointed styling for graceful degradation or `min-width` only and write mobile-first styles by default. It will be more consistent and you might avoid future confusion